api/services/Reddit.js

```diff
@@ -34,7 +35,7 @@ exports.refreshToken = async function(refreshToken) {
   }
 };
 
-var makeRequest = async function (refreshToken, requestType, url, data, rateLimitRemainingThreshold) {
+var makeRequest = async function (refreshToken, requestType, url, data, rateLimitRemainingThreshold, silentErrors) {
```

Comment:
When we attempt to lock an archived post, Reddit returns a 400 error. We can test for that case somewhere, but we don't need it to be logged as an error message, since in this case the 400 response is expected. So we turn on the silentErrors flag to squelch the error message.